### ISSUE TITLE
About I want to run the test set separately

### ISSUE THREAD

**Issue by GYQAme** [GuangxingHan/Meta-Faster-R-CNN#31]
How do you separate the training and test scripts without running the training script every time you test?

**Comment by GuangxingHan** [comment]
Please find the inference script [here](https://github.com/GuangxingHan/Meta-Faster-R-CNN/issues/2).
